docs/using-snippet.md

```diff
@@ -14,3 +14,13 @@ Requesting and downloading On Demand products can also be done programmatically:
 
 * [HyP3 SDK for Python](using/sdk.md "Using SDK")
 * [HyP3 REST API](using/api.md "Using API")
+
+### Public Visibility of Jobs
+
+Please note that all jobs submitted to HyP3, whether via web access or programmatic access, are publicly visible.
+This means that anyone with access to HyP3 can potentially:
+
+* View your jobs and associated metadata, including job name and user ID.
+* Download any products generated by your jobs.
+
+In particular, please do not include any sensitive information in the job name.
```

Comment:
```suggestion
!!! info Public Visibility of Jobs

    Please note that all jobs submitted to HyP3, whether via web access or programmatic access, are publicly visible.
    This means that anyone with access to HyP3 can potentially:

    * View your jobs and associated metadata, including job name and user ID.
    * Download any products generated by your jobs.

    In particular, please do not include any sensitive information in the job name.
```
What do you think about placing this inside an [admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#type:info)?

Reply:
I like it, but would we still be able to link to this section of the page like for e.g. https://hyp3-docs.asf.alaska.edu/#web-access? I was hoping to link to the section from the SDK's `HyP3` docstring to avoid having to re-summarize this information there.

Reply:
Pushed a compromise with both a header and an admonition
![Screenshot from 2023-05-15 15-33-58](https://github.com/ASFHyP3/hyp3-docs/assets/17994518/e688472a-c3f7-45e4-b604-82da018433e4)